raspiaudio-muse-luxe.yaml

```
@@ -83,34 +133,53 @@ binary_sensor:
       mode:
         input: true
         pullup: true
-    name: Volume Up
+    name: ${friendly_name} Volume Up
```

These are not needed because the device has a `friendly_name` and this will double it up
```suggestion
    name: Volume Up
```